srcs/requirements/frontend/src/navigation.ts

```
@@ -0,0 +1,172 @@
+
+/**
+ * @brief Navigates to a specified view.
+ * 
+ * This function updates the main application container with the content of the specified view's template.
+ * It also updates the browser history and sets up event listeners for the new view.
+ * 
+ * @param view The ID of the view to navigate to.
+ * @param replace A boolean indicating whether to replace the current history state or push a new one.
+ */
+
+export function navigateTo(view: string, replace: boolean = false): void {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)